parlai/agents/repeat_label/repeat_label.py

```diff
@@ -52,9 +51,4 @@ def act(self):
         else:
             reply['text'] = "I don't know."
 
-        if 'label_candidates' in obs and len(obs['label_candidates']) > 0:
```

Comment:
this was slow because label_candidates might not allow indexing (might be a set). we keep it in though.